### ISSUE TITLE
Can't locate Revert.pm 

### ISSUE THREAD

**Issue by pnorman** [woodpeck/osm-revert-scripts#5]
```
$ perl revert.pl 45635426 'Revert relation claiming mason-dixon line is a state'
Can't locate Revert.pm in @INC (you may need to install the Revert module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.24.1 /usr/local/share/perl/5.24.1 /usr/lib/x86_64-linux-gnu/perl5/5.24 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.24 /usr/share/perl/5.24 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at revert.pl line 8.
BEGIN failed--compilation aborted at revert.pl line 8.
```

**Comment by pnorman** [comment]
My fix was ``PERL5LIB=`pwd` perl revert.pl 45635426 'Revert relation claiming mason-dixon line is a state'``, but this isn't a great solution

**Comment by woodpeck** [comment]
Fixed in #4801750
